openeo/rest/job.py

```diff
@@ -9,7 +9,7 @@
 
 import requests
 
-from openeo.api.logs import LogEntry
+from openeo.api.logs import LogEntry, normalize_log_level, string_to_log_level
```

Comment:
That was bothering me too. I initially kept ``string_to_log_level`` because ``BatchJob.logs`` was calling it directly.
But that wasn't really necessary anymore.

I decided to eliminate ``string_to_log_level`` entirely because it helped to make the code simpler and cleaner overall.